README.md

```diff
@@ -40,6 +40,16 @@ confirmation = agent.submit_financials(
 confirmation.success? # => true
 confirmation.errors # A hash of errors if errors occur
 
+# Submit financials w/ a bank account
+confirmation = agent.submit_financials(
+  {
```

Comment:
super nitpick for style, I'd drop the `{`s here because they're optional for the last/only arg:

```ruby
confirmation = agent.submit_financials(
  checking_acct: '123456789',
  bank_routing: '987654321'
)
```